### PR TITLE
Created a local variable for ProxiedPlayers

### PR DESCRIPTION
**Pull request by terrarier2111**

### CODE REVIEW

**Comment by terrarier2111** [comment]
@KennyTV could you approve this changes please?

File: bungee/src/main/java/us/myles/ViaVersion/BungeePlugin.java [hunk 1]
```diff
@@ -126,9 +127,10 @@ public void cancelTask(TaskId taskId) {
 
     @Override
     public ViaCommandSender[] getOnlinePlayers() {
-        ViaCommandSender[] array = new ViaCommandSender[getProxy().getPlayers().size()];
+        Collection<ProxiedPlayer> players = getProxy().getPlayers();
+        ViaCommandSender[] array = new ViaCommandSender[players.size()];
```
Comment (mdcfe) [inline bungee/src/main/java/us/myles/ViaVersion/BungeePlugin.java:131]:
```suggestion
        ViaCommandSender[] array = new ViaCommandSender[getProxy().getPlayers().size()];
```

File: bungee/src/main/java/us/myles/ViaVersion/BungeePlugin.java [hunk 2]
```diff
@@ -126,9 +127,10 @@ public void cancelTask(TaskId taskId) {
...
         int i = 0;
-        for (ProxiedPlayer player : getProxy().getPlayers()) {
+        for (ProxiedPlayer player : players) {
```
Comment (mdcfe) [inline bungee/src/main/java/us/myles/ViaVersion/BungeePlugin.java:133]:
```suggestion
        for (ProxiedPlayer player : getProxy().getPlayers()) {
```

**Review by mdcfe — CHANGES_REQUESTED** [review]

File: bungee/src/main/java/us/myles/ViaVersion/BungeePlugin.java [hunk 3]
```diff
@@ -28,6 +28,7 @@
 
 import java.util.ArrayList;
 import java.util.Collections;
+import java.util.Collection;
```
Comment (mdcfe) [inline bungee/src/main/java/us/myles/ViaVersion/BungeePlugin.java:31]:
```suggestion
```

**Comment by terrarier2111** [comment]
You essentially told me to revert all changes i made is there any reason for this and if so could you please tell me the reason.

**Comment by FormallyMyles** [comment]
Thanks!

**Pull request merged**